package.json

```diff
@@ -38,6 +33,20 @@
   "devDependencies": {
     "babel-eslint": "3.1.20",
     "eslint": "0.24.0",
+    "eslint-config-universal-search": "1.0.0",
     "webpack-dev-server": "^1.10.1"
+  },
+  "homepage": "https://github.com/pdehaan/universal-search-content#readme",
+  "license": "MPL-2.0",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/pdehaan/universal-search-content.git"
+  },
```

Comment:
I ran `npm init` on this repo, and it added the repository info and other stuff. Otherwise you get some bonus warnings if you're using a newer version of npm (2.x or 3.alpha).